### PR TITLE
Feature: Add the textblockcomp

### PR DESCRIPTION
**Pull request by omeneses**
1. Link to Trello's task: https://trello.com/c/NYY3kKJM/16-mensaje-inicial-2
2. task's design in zeplin: https://app.zeplin.io/project/5f15bfe784a1689e60712353/screen/5f15c0228f1d6e9cee118eb9
3. Component's screenshot
Screen Shot 2020-08-01 at 11 13 55 PM
4. Code implemented using Grid.

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // Component imports
-import { defaultComp } from './components/default';
+import { defaultComp } from './components/default.js';
+import { initialMessageComp } from './components/initialmessage';
+
```
Comment (juan-ortizc) [inline src/js/index.js]: Please use camalcase to name the JS files 

'initialMessage'

File: src/js/components/initialmessage.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/**
+ * @param {object} data contains the expected page content data
+ * returns a default component to show the provided information.
+ */
+const initialMessageComp = (data) => {
+  const section = document.createElement('SECTION');
+  
+  if (data.backgroundColor) section.classList.add(`textblockcomp__container--${data.backgroundColor}`);
```
Comment (juan-ortizc) [inline src/js/components/initialmessage.js]: Please use - to separate compound words in classes

'text-block-comp'

File: src/js/components/initialmessage.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  
+  if (data.title){
+    let title = document.createElement('H1');
+    title.classList.add('textblockcomp__title');
```
Comment (juan-ortizc) [inline src/js/components/initialmessage.js]: Same here

File: src/js/components/initialmessage.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    title.textContent = data.title;
+    section.appendChild(title);
+  }
+  
+  if (data.text){
+    let text = document.createElement('P');
+    text.classList.add('textblockcomp__text');
```
Comment (juan-ortizc) [inline src/js/components/initialmessage.js]: Same here

File: src/js/components/initialmessage.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    text.textContent = data.text;
+    section.appendChild(text);
+  }
+  
+  section.classList.add('textblockcomp__container');
```
Comment (juan-ortizc) [inline src/js/components/initialmessage.js]: Same here

File: src/scss/main.scss [hunk 6]
```diff
@@ -8,3 +8,7 @@
 // 2. Base stuff
 @import 'base/reset';
 @import 'base/typography';
+@import 'base/grid';
+
+// 3. Styling components
+@import 'components/textblockcomp'
```
Comment (juan-ortizc) [inline src/scss/main.scss]: Use - to separate compound words in SCSS files

'text-block-comp

File: src/js/index.js (hunk 1 above)
Comment (omeneses) [reply src/js/index.js]: Done!!!  camelCase implemented.

File: src/js/components/initialmessage.js (hunk 2 above)
Comment (omeneses) [reply src/js/components/initialmessage.js]: Done!!!

File: src/js/components/initialmessage.js (hunk 3 above)
Comment (omeneses) [reply src/js/components/initialmessage.js]: Done!!!

File: src/js/components/initialmessage.js (hunk 4 above)
Comment (omeneses) [reply src/js/components/initialmessage.js]: Done!!!

File: src/js/components/initialmessage.js (hunk 5 above)
Comment (omeneses) [reply src/js/components/initialmessage.js]: Done!!!

File: src/scss/main.scss (hunk 6 above)
Comment (omeneses) [reply src/scss/main.scss]: Done!!!

**Review by juan-ortizc — APPROVED** [review]
well done

**Review by ArleyAV — APPROVED** [review]
Well done Omar!

**Comment by jsebax** [comment]
Please fix merge conflicts

File: src/js/components/initialmessage.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
+/**
+ * @param {object} data contains the expected page content data
+ * returns a default component to show the provided information.
+ */
+const initialMessageComp = (data) => {
+  const section = document.createElement('section');
+  
+  if (data.backgroundColor) section.classList.add(`text-block-comp__container--${data.backgroundColor}`);
```
Comment (jsebax) [inline src/js/components/initialmessage.js]: This component does not have a background color variation. The green box with the white color section is going to be handled in a different ticket.

Comment (omeneses) [reply src/js/components/initialmessage.js]: Done!!!

**Pull request merged**